proxy.go

```diff
@@ -191,6 +191,11 @@ func main() {
 
 	flag.Parse()
 
+	if len(channel) == 0 || len(proxyAddr) == 0 {
```

Comment:
Thanks for raising! I'd simplify this to the following though:

```
if channel == "" || proxyAddr == "" {
    ...
}
```

In fact, you could also change the first check in `unixAddr()` to do the same.